src/Microsoft.ML.OnnxTransform/OnnxUtils.cs

```diff
@@ -0,0 +1,302 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Scoring;
+using OnnxShape = System.Collections.Generic.List<long>;
+
+namespace Microsoft.ML.OnnxScoring
+{
+
+    /// <summary>
+    /// OnnxModel is a facad for ModelManager. ModelManager is provided by Sonoma API,
+    /// and it has a lot of functionality (multiple models, multiple versions) that are not
+    /// needed by Onnx transform, which only needs a single model. This facad simplifies the
+    /// usage of onnx model.
+    /// </summary>
+    internal sealed class OnnxModel
```

Comment:
>OnnxModel [](start = 26, length = 9)

might consider separate file for OnnxModel.  

misnomer to put it in same file as OnnxUtils.cs (which has all static members)

Reply:
OnnxModel is also a util for deficiencies in sonoma ModelManager api. It should go away once we have proper APIs from lotus.

Reply:
OnnxModel is also a util for deficiencies in sonoma ModelManager api. It should go away once we have proper APIs from lotus.

---
In reply to: [219918823](https://github.com/dotnet/machinelearning/pull/942#discussion_r219918823) [](ancestors = 219918823)